### DISCUSSION TITLE
6月17号付款后页面直接关闭，未跳转指引如何使用。授权邮箱使用，也未收到验证码相关邮件，垃圾邮箱里也没有

### DISCUSSION THREAD

**Discussion by Z199-Hai** [getlantern/lantern-forum-cn#614] [General]
![微信图片_20240617165039](https://github.com/getlantern/lantern-forum-cn/assets/173046878/c715e4ac-85be-41d5-9dfb-1f8e56331ebe)

**Comment by Z199-Hai** [comment]
@Mike777Khoo

**Reply by MaxMKMax** [reply]
@Z199-Hai 您好. 我帮您提交问题. 请您也提供我您的账号邮箱.

**Reply by Z199-Hai** [reply]
> @Z199-Hai 您好. 我帮您提交问题. 请您也提供我您的账号邮箱.

@Mike777Khoo 你好，我的邮箱是 19924252871@163.com  请尽快处理，急用！！

**Comment by Z199-Hai** [comment]
@Mike777Khoo 你好，我的邮箱是 19924252871@163.com 请尽快处理，急用！！！
